src/Compilers/Core/MSBuildTask/Shared/ManagedCompiler.cs

```diff
@@ -689,6 +687,14 @@ protected internal virtual void AddResponseFileCommands(CommandLineBuilderExtens
             commandLine.AppendFileNamesIfNotNull(Sources, " ");
         }
 
+        internal void AddResponseFileCommandsForSwitchesSinceInitialReleaseThatAreNeededByTheHost(CommandLineBuilderExtension commandLine)
```

Comment:
The louder I can make the "you should put that there" I can make this, I will. If Visual Studio supported a blink tag, I'd do that. :-)